.gitignore

```diff
@@ -24,7 +24,7 @@ testbin/*
 *.swp
 *.swo
 *~
-kuadrant-operator
+/kuadrant-operator
```

Comment:
My only critique is that having a folder with the same repo name at the root of the repo can seem a little confusing for development. You mentioned binaries being compiled here... is there any way to use a `bin` directory ?

Reply:
I can't remember which target/script will trigger the build of it, it's the same for limitador operator, you can take a look at https://github.com/Kuadrant/limitador-operator/blob/main/.gitignore#L9, coming from https://github.com/Kuadrant/limitador-operator/commit/18a255fcf7f335c38b4f0a4c09ff916bc81eb88a

Reply:
OK, since it appears to be the de facto way of handling the binaries I will resolve for now. Thanks for the additional context. :)